tests/test_multicast_expert.py

```diff
@@ -340,4 +340,43 @@ def test_v6_loopback_multiple() -> None:
                     print("\nRx: " + repr(packet_alt))
                     assert packet_alt is not None
                     assert packet_alt[0] == test_string_alternate
-                    assert packet_alt[1][0:2] == (multicast_expert.LOCALHOST_IPV6, mcast_tx_sock_alt.getsockname()[1])
\ No newline at end of file
+                    assert packet_alt[1][0:2] == (multicast_expert.LOCALHOST_IPV6, mcast_tx_sock_alt.getsockname()[1])
+
+
+def test_external_loopback_v4() -> None:
+    """
+    Check that packets sent over external interface can be received when `enable_external_loopback` is set.
```

Comment:
Can we also add a test that verifies that when loopback is set to false, packets are not received by the rx socket?

Reply:
Added test